index.html

```diff
@@ -5,11 +5,92 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
+    <header>
+        <div class="image-frame">
+            <img class="img-header" src="img/photo-cake3.jpg" alt="Header Photo">
+        </div>
+        <div class="right-header">
+            <div  class="menu-frame">
+                <img  class="menu-header" src="img/menu-hamburger.svg" alt="Menu">
+                <div class="div-free">
+                    <p></p>
+                </div>
+            </div>
+            <article class="header-text">
+                Treat yourself to the sweetest and most delicious taste.
+            </article>
+        </div>
+    </header>
+    <main>
+        <nav class="the-nav">
+            <ul class="un-order-list">
+                <div class="free-li-dev"></div>
+                <div class="rectangle">
+                    <li>HOME</li>
+                </div>
+                <div class="rectangle">
+                    <li>CAKES</li>
+                </div>
+                <div class="rectangle">
+                    <li>ORDERIN</li>
+                </div>
+                <div class="rectangle">
+                    <li>LESSONS</li>
+                </div>
+                <div class="rectangle">
+                    <li>ABOUT</li>
+                </div>
+            </ul>
+        </nav>
+        <div class="new-order-main">
+            <div class="main-right">    
+                <article class="welcome">
+                   <h2>Welcome</h2>
+                </article>
+                <article class="main-text">
+                  <p>Our cakes are made from the finest quality flour and sugar and do not contain any preservatives.
+                     We have been making cakes for generations, offering high service and fine taste.
+                     A wide variety of cakes that are suitable for all occasions.
+                     We also cater to special requests and new styles that meet customers' desires.</p>
+               </article>
+            </div>
+            <div class="div-main-img">
+               <img class="main-image" src="img/photo-cake1.jpg" alt="Main image">
```

Comment:
Your `alt` attribute should describe what is shown in your image. "Main image" is not a very helpful description! 😄

Reply:
Indeed, you are using the same `alt` text for all your images